### PR TITLE
Configure if ip is displayed v2

### PR DESCRIPTION
**Pull request by boehla**
Make configurable to display the ip address

### CODE REVIEW

File: html/template.html [hunk 1]
```diff
@@ -71,7 +71,9 @@
 <?php if (isset($config['display_testnet']) && $config['display_testnet']) { ?>
       <p class="stat"><span class="label">Testnet:</span> <?php echo ($data['testnet']) ? "Yes" : "No"; ?></p>
 <?php } ?>	 
-      <p class="stat"><span class="label">Node IP Address:</span> <?php echo $data['node_ip']; ?></p>
+<?php if(isset($config['display_ip']) && $config['display_ip']) { ?>
```
Comment (craigwatson) [inline html/template.html]: Can you fix the Travis CI failure here? Just adding a space after the `if` should fix it :+1:

**Comment by craigwatson** [comment]
Thanks very much for the PR - only one minor niggle with syntax (I use the PEAR PHP standards).

Will merge once Travis is happy, code looks good though :+1:

File: html/template.html (hunk 1 above)
Comment (boehla) [inline html/template.html]: Aaah. This was the error.
Then I have to make a new pull? Or I'm only unable to find this feature.

Comment (craigwatson) [inline html/template.html]: You can just commit to your master and it will show up on this pull request :)

Comment (boehla) [inline html/template.html]: Ok i have done it, but there seems to be an other problem then the spaces ?.. :S

**Comment by boehla** [comment]
Ok I have put an space after 'if' but there is still a problem.
Do you have an idea?

**Comment by craigwatson** [comment]
Apologies for the late reply - as per the error on Travis CI, you'll need to add a space at the end of the 'if' statement, between the closing parenthesis and the opening curly-brace.

Full syntax should be:

```
if ($var == "somevalue") {
    ...
}
```

**Comment by boehla** [comment]
Yeah, ok, good to know ;) Very thanks.
Ready to merge my first pull ;)

**Comment by craigwatson** [comment]
Merged, congratulations :+1:

**Pull request merged**